controllers/storagecluster/initialization_reconciler_test.go

```diff
@@ -163,8 +157,8 @@ func createFakeInitializationStorageClusterReconciler(t *testing.T, obj ...runti
 func createFakeInitializationStorageClusterReconcilerWithPlatform(t *testing.T,
 	platform *Platform,
 	obj ...runtime.Object) StorageClusterReconciler {
-	scheme := createFakeInitializationScheme(t, obj...)
-	obj = append(obj, mockNodeList)
+	scheme := createFakeScheme(t)
```

Comment:
Earlier this scheme had a lot of api schemes added. Is it still the case or it's just the default scheme?

Reply:
yes, this scheme has a lot of api schemes.  `createFakeInitializationScheme()` also had a lot of api schemes added, which was duplicated.